android/build.gradle

```diff
@@ -30,6 +30,6 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.19.+'
+    compile 'com.facebook.react:react-native:0.32.+'
```

Comment:
Hi I have my app with RN 0.33 and didn't face any such problem after implementing PR changes along with `compile 'com.facebook.react:react-native:+`